build/commands/scripts/mcp-setup.js

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const { execSync, spawn } = require('child_process');
+const chalk = require('chalk');
+
+const isWindows = process.platform === 'win32';
+const isMacOS = process.platform === 'darwin';
+const isLinux = process.platform === 'linux';
+
+const PROJECT_ROOT = path.resolve(__dirname, '../../../../..');
+const BRAVE_CORE_ROOT = path.resolve(__dirname, '../../..');
+const CLAUDE_CONFIG_PATH = path.join(PROJECT_ROOT, '.claude', 'mcp_servers.json');
+const CURSOR_CONFIG_PATH = path.join(require('os').homedir(), '.cursor', 'mcp.json');
+const GLOBAL_CLAUDE_CONFIG_PATH = path.join(require('os').homedir(), '.claude', 'mcp_servers.json');
+const REQUIREMENTS_PATH = path.join(BRAVE_CORE_ROOT, 'agents', 'mcp', 'requirements.txt');
+
+function log(message, type = 'info') {
+  const colors = {
+    info: chalk.blue,
+    success: chalk.green,
+    warning: chalk.yellow,
+    error: chalk.red
+  };
+  console.log(colors[type](`[MCP Setup] ${message}`));
+}
+
+function ensureDirectoryExists(dirPath) {
+  if (!fs.existsSync(dirPath)) {
+    fs.mkdirSync(dirPath, { recursive: true });
+    log(`Created directory: ${dirPath}`, 'success');
+  }
+}
+
+function getPythonCommand() {
+  // Try to find the best Python command for the platform
+  const pythonCommands = ['python3', 'python'];
```

Comment:
nit: this should just be set to the default used for others. I think this is python3